dso-l2/src/main/java/com/tc/objectserver/impl/DiagnosticsHandler.java

```diff
@@ -95,11 +95,7 @@ private void processMessage(TCMessage message) {
           result = initialState != null ? initialState.getName().getBytes(set) : "".getBytes(set);
           break;
         case "getClusterState":
-          if (GuardianContext.validate(Guardian.Op.SERVER_DUMP, "getClusterState")) {
-            result = server.getClusterState(set, null);
-          } else {
-            result = "NOT PERMITTED".getBytes(set);
-          }
+          result = server.getClusterState(set, null);
```

Comment:
This operation wasn't a dump operation - it was "get me the dumps" operation, something we don't need to guard (and thus audit)

Reply:
The intention of auditing cluster tool dump was to avoid abuse of server logging (somebody could fill up the server logs with endless backup data). The analogous action from TMC is to fetch the server dumps in the browser and view/save them. This operation is a non-privileged operation in TMC, and was never being audited. I, thus, removed this guard, especially that `SERVER_DUMP` operation is gone.